### ISSUE TITLE
[Accessibility] Inability to Interact with "Saved Filter" under map view for Keyboard-Only Users 

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4967]
## Describe the bug

* The saved filter in the "Map View" is not keyboard-focusable and operable. Users are unable to choose from the saved filters by ticking the checkboxes using the keyboard

## To Reproduce

1. Launch the local host
2. Navigate to the "Map View"
3. Find the saved filters under the Map View and try to interact with it without using the mouse.
4. Observe if you are able to complete these actions using the keyboard only.

## Expected behavior

Saved filter should be keyboard-focusable and operable, allowing users to interact with it

## Screenshots

![Screenshot from 2024-06-18 17-58-07](https://uploads.linear.app/f1df6e34-3bca-4ade-bb48-e6a457e64091/afc929e4-3160-448e-8fbe-08f3dc57c922/62880b28-7a5f-4005-9373-469148d5bdd3?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2YxZGY2ZTM0LTNiY2EtNGFkZS1iYjQ4LWU2YTQ1N2U2NDA5MS9hZmM5MjllNC0zMTYwLTQ0OGUtOGZiZS0wOGYzZGM1N2M5MjIvNjI4ODBiMjgtN2E1Zi00MDA1LTkzNzMtNDY5MTQ4ZDViZGQzIiwiaWF0IjoxNzE4OTYzMzYyLCJleHAiOjMzMjg5NTIzMzYyfQ.171YNbvnhO5zLEgPe4bbhUlC5Mt4XsObr-7LWCtZRyg)

**Comment by Angamanga** [comment]
Closing but keeping in Linear
